static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -80,8 +81,177 @@ define([
         return this.addMessageVariant(variantId, {contributions: variantParams});
     };
 
-    return [new EditionTest('UK', 'MembershipEngagementBannerUkRemindMeLater', '2017-02-02', '2017-02-28', 'remind_me_later')
-        .addMembershipVariant('control', {})
-        .addMembershipVariant('remind_me', {showRemindMe : true})
-    ];
+    // Below are the 6 messages that are being tested, modulo the currency/amount which is dependent on the user's geolocation.
+    // The reader's geolocation is also used on the membership website to determine the currency/amount,
+    // therefore, the pricing should remain consistent between the engagement banner and the membership website.
+    //
+    // CONTROL
+    // For less than the price of a coffee a week, you could help secure the Guardian's future.
+    // Support our journalism for £5 a month.
+    //
+    // WEEKLY PRICE
+    // For less than the price of a coffee a week, you could help secure the Guardian's future.
+    // Support our journalism from 95p a week.
+    //
+    // SOCIAL PROOF
+    // Join hundreds of thousands of Guardian readers and help secure our future.
+    // Support our journalism today for just £5 a month.
+    //
+    // OPPORTUNITY
+    // If you’ve been thinking about supporting us, we’ve never needed you more.
+    // Support our journalism today for just £5 a month.
+    //
+    // SPEED
+    // It only takes two minutes to play your part in helping to secure the Guardian’s future.
+    // Support our journalism today for just £5 a month.
+    //
+    // LOW COST / BIG IMPACT
+    // From helping us hold power to account, to giving a voice to the voiceless, your money can do a lot of good.
+    // Support our journalism for just £5 a month.
+
+    // We don't want to run these tests on the US or Australia audience, as this would delay releasing them by ~ week.
+    function isNotInUSOrAU() {
+        var countryCode = geolocation.getSync();
+        return countryCode !== 'US' && countryCode !== 'AU'
+    }
+
+    // Prices taken from https://membership.theguardian.com/<region>/supporter
+    var monthlySupporterCost = {
+        GB:  '£5',
+        US:  'US$6.99',
+        AU:  'AU$10',
```

Comment:
Hey, had a bit more of a think regarding this since we spoke. Since the test isn't running in the US or AUS, `getSupporterPaymentRegion()` should never return `US` or `AU`, but in the instance it does, I think it makes sense to use the local cost. When the test is released, I'll use the `pageviews` table to determine whether this is happening at all. Is that ok?